programmers/제일-작은-수-제거하기.js

```diff
@@ -0,0 +1,5 @@
+function solution(arr) {
+    if(arr.length > 1) {
+        return arr.filter((x) => x !== Math.min.apply(null, arr));
+    }return [-1];
+}
```

Comment:
```suggestion
function solution(arr) {
  if (arr.length > 1) {
    return arr.filter((x) => x !== Math.min(...arr));
  }
  return [-1];
}
```
1. 컨벤션 확인
2. 변수명 확인
3. console.log() 지웠는지 확인
4. spread연산자를 사용하면 apply가 필요없답니다 ~ 두가지방법 다 알면 좋아요 ~

```js
function solution(arr) {
  arr.splice(arr.indexOf(Math.min(...arr)), 1);
    
  return arr.length ? arr : [-1] 
}
```
이건 제가 푼 방식입니다요 ~